src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,47 @@
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo,
+  deleteTodo: (id: number) => void,
+  processedIds: number[],
```

Comment:
The 'processedIds' prop is not explained or used. Consider removing it if it's not necessary